### DISCUSSION TITLE
Developing a new navbar component.

### DISCUSSION THREAD

**Discussion by sanketshevkar** [chakra-ui/chakra-ui#4161] [Ideas] [answered]
Wanted to know whether if adding a new navbar component to the library would be good idea?

**Comment by anubra266** [comment, marked as answer]
https://choc-ui.tech/docs/navigation/navbars
https://chakra-templates.dev/navigation/navbar

**Reply by eric-hc** [reply]
Also, there is [Chakra Pro](https://pro.chakra-ui.com/), created by the Chakra team

**Comment by with-heart** [comment]
This is best left as a userland component, similar to the navbars offered by Choc UI and Chakra Templates. Chakra is focused on providing low-level, accessible building blocks that can be used to build more high-level components like a navbar.

**Comment by sanketshevkar** [comment]
Okay got it. Thank you @anubra266 and @with-heart !
